hapic/ext/aiohttp/context.py

```diff
@@ -133,6 +134,17 @@ def get_request_parameters(self, *args, **kwargs) -> RequestParameters:
 
         raise WorkflowException("Unable to get aiohttp request object")
 
+    def get_file_response(
+        self,
+        file_response: HapicFile,
+        http_code: int
+    ) -> "Response":
+        if file_response.file_path:
```

Comment:
Don't forgot to test all case of ̀HapicFile` before MR opening.

Reply:
Done.